warp-contracts-plugin-signature/src/web/evm/common.ts

```diff
@@ -0,0 +1,55 @@
+import Transaction, { Tag } from 'arweave/web/lib/transaction';
+import { encodeTxId } from '../utils';
+import { stringify } from 'safe-stable-stringify';
+import { SmartWeaveTags, TagsParser } from 'warp-contracts/web';
+import { Interaction } from './evmSignatureVerification';
+
+export const prepareEvmTransaction = (tx: Transaction, ownerAddress: string) => {
```

Comment:
i was even thinking of creating `common.ts` file under `web` folder as each of the wallets implementation will be preparing the transaction exactly the same way, so it wont be `prepareEvmTransaction` but `prepareTransaction`. In this file only `attachTags` method should be parametrized with a dedicated signature type.

Reply:
Good point!

Reply:
However, more thinking from user :)

Reply:
Done, as you said 📦